### PR TITLE
Add Solr resize API to R&R

### PR DESCRIPTION
**Pull request by anathashavit**
Add a cluster resize API that allows posting a resize operation and getting information on a cluster's size using the Retrieve&Rank service.  
In addition there is an example file that shows how to use the new API. 

Thanks for contributing to the Watson Developer Cloud!

### CODE REVIEW

File: src/main/java/com/ibm/watson/developer_cloud/retrieve_and_rank/v1/model/SolrClusterResizeRequest.java [hunk 1]
```diff
@@ -0,0 +1,42 @@
+/*
+ * BEGIN_COPYRIGHT
```
Comment (germanattanasio) [inline src/main/java/com/ibm/watson/developer_cloud/retrieve_and_rank/v1/model/SolrClusterResizeRequest.java]: The headers should make the ones we use in the project

**Comment by germanattanasio** [comment]
Can we add unit tests to the new functionality ?

File: src/main/java/com/ibm/watson/developer_cloud/retrieve_and_rank/v1/model/SolrClusterResizeRequest.java (hunk 1 above)
Comment (anathashavit) [inline src/main/java/com/ibm/watson/developer_cloud/retrieve_and_rank/v1/model/SolrClusterResizeRequest.java]: Done

**Comment by germanattanasio** [comment]
@anathashavit the Retrieve and Rank integration test will take a lot of time. Can you add a unit test to make sure the request match the API specs? You can take a look at the other unit tests if you need.

**Comment by anathashavit** [comment]
@germanattanasio looking at the R&R test package I see that all other API's are testes the same way. 
There are a couple of unit tests for utility classes used by the API but not foe the API itself since it would be very difficult to verify the API using only a unit test.

**Comment by germanattanasio** [comment]
What I'm asking is something like:

```java
    // mock api call with a default json response
    ToneAnalysis mockResponse = loadFixture(FIXTURE, ToneAnalysis.class);
    server.enqueue(jsonResponse(mockResponse));
    
    // execute request
    ToneAnalysis serviceResponse = service.getTone(text, null).execute();

    RecordedRequest request = server.takeRequest();

    assertEquals(TONE_PATH, request.getPath());
    assertNotNull(request.getHeader(HttpHeaders.AUTHORIZATION));
    assertEquals(serviceResponse, mockResponse);
    assertEquals(HttpMediaType.APPLICATION_JSON, request.getHeader(HttpHeaders.ACCEPT));
```
Take a look at the unit tests for [Tone Analyzer](https://github.com/watson-developer-cloud/java-sdk/blob/master/src/test/java/com/ibm/watson/developer_cloud/tone_analyzer/v3/ToneAnalyzerTest.java)

The APIs that you are wrapping in `resizeSolrCluster` and `getSolrClusterResizeStatus` should be tested with a unit test.

**Pull request merged**